elip-0100.mediawiki

```diff
@@ -64,15 +64,17 @@ This specification relies on definition of roles in the main PSET standard. The
 
 As a single entity is likely to be both a Creator and Updater, and thus it makes possible to fill-in asset metadata on PSET creation step.
 
+<tt>WARNING:</tt> PSET containing asset metadata is not supposed to be transmitted to other parties to not encourage them to trust this information. Updater must remove all asset metadata records after completing the signature process involving an offline entity.
+
 ===Signer===
 
 In addition to the Signer behavior defined in BIP 370 PSBT and the main PSET standard, signer should be responsible for properly handling fields belonging to Hardware Wallet Extensions. Within the scope of this specification, it is assumed that the Signer role is implemented by a hardware wallet or any application running on an airgapped device with similar concept.
 
-Asset metadata fields, <tt>PSBT_ELEMENTS_HWW_GLOBAL_ASSET_METADATA</tt> are fully optional, meaning that the Signer should not assume their presence for all or some of the assets referenced in the transaction. But if any of the asset metadata fields is present, the Signer must verify it by re-computing the asset tag from the given contract and issuance transaction's input. This field is provided to the Signer for displaying rich asset information to the user. Since a hardware wallet typically does not have access to the network, PSET fields may be the only source of metadata for this category of Signer implementations.
+Asset metadata fields, <tt>PSBT_ELEMENTS_HWW_GLOBAL_ASSET_METADATA</tt> are fully optional, meaning that the Signer should not assume their presence for all or some of the assets referenced in the transaction. But if any of the asset metadata fields is present, the Signer must verify it by re-computing the asset tag from the given contract and issuance transaction's input. This field is provided to the Signer for displaying rich asset information to the user. For security reasons, it is highly recommended that externally provided asset metadata is displayed along with the full asset tag in hexadecimal form, if it is within technical capabilities of the HWW. Since a hardware wallet typically does not have access to the network, PSET fields may be the only source of metadata for this category of Signer implementations, except for its internal asset database.
```

Comment:
I suggest:

`if it is within technical capabilities of the HWW` -> `if it is within technical capabilities of the HWW. This is because the asset metadata may be be valid, but contain e.g. a ticker that mimics another asset. If only the ticker is shown, the user signing the transaction may be tricked into signing an asset that they did not intend.`